models/dns.go

```diff
@@ -59,6 +59,19 @@ func StringsToNameservers(nss []string) []*Nameserver {
 	return nservers
 }
 
+// NameserversToStrings constructs a list of lists from *Nameserver structs
+func NameserversToStrings(nss []*Nameserver) (s []string) {
+	for _, ns := range nss {
+		s = append(s, ns.Name)
+	}
+	return s
+}
+
+// FIXME(tal): In hindsight, the Nameserver struct is overkill. We
```

Comment:
Lets remove this comment since there is an issue to track this now.

Reply:
I couldn't find the issue, so I created one. https://github.com/StackExchange/dnscontrol/issues/577

I've updated the comment to link to the issue. (as is the Google Styleguide standard... if you list a FIXME(), 90% of the time there should be an issue filed)